dyno-core/src/main/java/com/netflix/dyno/connectionpool/ConnectionPool.java

```diff
@@ -148,4 +148,6 @@
      */
     HealthTracker<CL> getHealthTracker();
 
+    boolean isIdle();
```

Comment:
That method is on the ConnectionPool interface so it doesn't make sense to name it specifically for a shadow implementor.